cmd/kube-apiserver/app/options/options_test.go

```diff
@@ -293,6 +294,9 @@ func TestAddFlags(t *testing.T) {
 		APIEnablement: &apiserveroptions.APIEnablementOptions{
 			RuntimeConfig: cliflag.ConfigurationMap{},
 		},
+		NetworkProxy: &apiserveroptions.NetworkProxyOptions{
+			ConfigFile: "/var/run/kubernetes/network-proxy/connectivity.yaml",
```

Comment:
is this anything standard? Why do we hardcode this here?

Reply:
This is just a value for testing.